app/src/main/java/com/example/taskyy/di/AuthModule.kt

```diff
@@ -33,6 +33,7 @@ object AuthModule {
     @Provides
     @Singleton
     fun provideUserDao(): UserDao {
+        //this doesn't work since it needs the composable annotation. How to get the context here?
         return TaskyyDatabase.getDatabase(LocalContext.current).userDao()
     }
```

Comment:
You're not in a composable function, which is why this doesn't work. With Dagger, you can do this:
```
@Provides
@Singleton
fun provideUserDao(@ApplicationContext context: Context): UserDao {
    return TaskyyDatabase.getDatabase(context).userDao()
}
```